rapid/src/test/java/com/vrg/rapid/integration/AbstractIT.java

```diff
@@ -0,0 +1,241 @@
+package com.vrg.rapid.integration;
+
+import lombok.Getter;
+import lombok.experimental.Accessors;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+
+import java.lang.reflect.Field;
+import java.nio.file.Paths;
+
+import static java.nio.file.Files.write;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.UUID;
+import java.util.concurrent.Executors;
+
+import com.google.common.io.Files;
+
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.Assert;
+
+/**
+ * AbstractIT for running rapid agents.
+ * <p>
+ * Created by zlokhandwala on 5/30/17.
+ */
+public class AbstractIT {
+
+    private static String RAPID_PROJECT_DIR = new File("..").getAbsolutePath() + File.separator;
+    private static String RAPID_RUNNER_JAR = RAPID_PROJECT_DIR + "rapid-examples/target/rapid-examples-1.0-SNAPSHOT-allinone.jar";
```

Comment:
Is there a way to not hard code this? Having the rapid code-base have a dependency on rapid-examples feels odd. Maybe have something similar to the standalone agent, but without the akka related boilerplate? And this poses problems for doing an "mvn clean package", because rapid-examples/ is built after rapid/.